### PR TITLE
Fix of some nGraph python tests

### PR DESCRIPTION
**Pull request by tomdol**
### Details:
 - Returning a single value if a test execution calculated a scalar (runtime.py)
 - Fix of the incorrect values in the softmax test
 - correct opset version for tests with "broadcast" attribute

### Tickets:
 - *44970*
 - *35925*

### CODE REVIEW

File: ngraph/python/tests/test_ngraph/test_ops_unary.py [hunk 1]
```diff
@@ -100,14 +99,13 @@ def sigmoid(x):
...
 
-@xfail_issue_44970
 def test_softmax():
-    axis = 0
+    axis = 1
     input_tensor = np.array([[1, 2, 3], [4, 5, 6]], dtype=np.float32)
 
     result = run_op_node([input_tensor], ng.softmax, axis)
 
-    expected = [[0.00426978, 0.01160646, 0.03154963], [0.08576079, 0.23312202, 0.6336913]]
```
Comment (tsocha) [inline ngraph/python/tests/test_ngraph/test_ops_unary.py:110]: it is a different test now, I wonder if it can be called a fix for issue.
Why did you change axis and results?

**Review by tsocha — CHANGES_REQUESTED** [review]

File: ngraph/python/tests/test_ngraph/test_ops_unary.py (hunk 1 above)
Comment (tomdol) [reply ngraph/python/tests/test_ngraph/test_ops_unary.py:110]: The previous results were incorrect for `axis == 0`. The correct values are:
```
[[0.04742587, 0.04742587, 0.04742587],
 [0.95257413, 0.95257413, 0.95257413]]
```
I've changed both the axis and correct expected values in this test.

**Review by tsocha — APPROVED** [review]

**Review by postrational — APPROVED** [review]
